libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/CriJourneyRequest.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.di.ipv.core.library.domain;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@ExcludeFromGeneratedCoverageReport
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+@Data
+public class CriJourneyRequest extends JourneyRequest {
+    private String language;
+
+    @Builder(builderMethodName = "criJourneyRequestBuilder")
```

Comment:
Not sure if we need to have a custom builder name here as it's only ever used as `CriJourneyRequest.builder()` so it's clear what type it is.

Reply:
Ah I saw when referencing builder, it references the JourneyRequest builder, which didn't work. But I see by keeping the annotation without the explicit naming it overrides the builder and does work